feature/system/attestz/attestz/tests/README.md

```diff
@@ -0,0 +1,78 @@
+   - IAK and IDevID cert contain the same device serial number field.
+4. Verify that the device returned the correct `ControlCardVendorId` with all fields populated.
+5. Issue owner IAK (oIAK) and owner IDevID (oIDevID) certs, which are based on the same underlying public keys, have the same structure and fields, but are signed by a different - owner - CA.
+6. Call `RotateOIakCert` to store newly issued oIAK and oIDevID certs and verify successful response.
+7. Call `GetIakCert` for a standby control card with correct `ControlCardSelection`.
+8. Repeat step (2) (TLS session will be secured by active control card's IDevID) and verify IDevID cert of standby control card was specified in the response payload.
+9. Repeat steps (3-6) for the standby control card.
+10. Call `Attest` for active control card with correct `ControlCardSelection`, random nonce, hash algo of choice (all should be supported and tested) and all PCR indices.
+11. Verify that the correct oIDevID cert was used for establishing TLS session.
+12. Verify that the device returned the correct `ControlCardVendorId` with all fields populated.
+13. Verify oIAK cert is the same as the one installed earlier.
+14. Verify all `pcr_values` match expectations.
+15. Verify `quote_signature` signature with oIAK cert.
+16. Use `pcr_values` and `tpms_quote_info` to recompute PCR Quote digest and verify that it matches the one used in `quote_signature`.
+17. Call `Attest` for standby control card with correct `ControlCardSelection`, random nonce, hash algo of choice (all should be supported and tested) and all PCR indices.
+18. Verify that the oIDevID cert of active control card was used for establishing TLS session and verify that oIDevID cert of standby control card was specified in the response payload.
+19. Repeat steps (12-16) for the standby control card.
+
+### Successful oIAK/oIDevID rotation
+
+1. Execute "Initial Install" workflow.
+2. Issue new oIAK and oIDevID certs for active control card, call `RotateOIakCert` to store those on the right card and verify successful response.
+3. Issue new oIAK and oIDevID certs for standby control card, call `RotateOIakCert` to store those on the right card and verify successful response.
+4. Call `Attest` for active and standby control cards and ensure they use the latest oIAK for attestation and, if there is no owner-provisioned TLS cert installed, use latest oIDevID for TLS session (both oIDevID-based and mTLS-cert-based scenarios should be tested).
+5. Do the same verification of attestation responses as in "Initial Install" workflow.
+
+### Successful post-install re-attestation
+
+1. Execute "Initial Install" workflow.
+2. Provision the device with switch owner mTLS credentials.
+3. Call `Attest` for active and standby control cards and ensure they use the new mTLS cert for TLS connection and the latest oIAK for attestation.
+4. Do the same verification of attestation responses as in "Initial Install" workflow.
+
+### Failure Scenarios
```

Comment:
each of these should be captured as test

Reply:
https://github.com/openconfig/featureprofiles/tree/main/feature/system/bootz/tests